### DISCUSSION TITLE
index.js:1 Warning: routes 将会废弃，为了保证兼容请使用 children 作为子节点定义方式 console.<computed> @ index.js:1

### DISCUSSION THREAD

**Discussion by zhangxinyong12** [umijs/umi#10825] [Q&A]
请问这个是怎么回事？
`{
  "name": "ant-design-pro",
  "version": "6.0.0-beta.1",
  "private": true,
  "description": "An out-of-box UI solution for enterprise applications",
  "scripts": {
    "analyze": "cross-env ANALYZE=1 max build",
    "build": "max build && node build-dist-zip.js",
    "build:prod": "max build",
    "deploy": "npm run build && npm run gh-pages",
    "dev": "npm run start:dev",
    "gh-pages": "gh-pages -d dist",
    "i18n-remove": "pro i18n-remove --locale=zh-CN --write",
    "postinstall": "max setup",
    "lint": "npm run lint:js && npm run lint:prettier && npm run tsc",
    "lint-staged": "lint-staged",
    "lint-staged:js": "eslint --ext .js,.jsx,.ts,.tsx ",
    "lint:fix": "eslint --fix --cache --ext .js,.jsx,.ts,.tsx --format=pretty ./src ",
    "lint:js": "eslint --cache --ext .js,.jsx,.ts,.tsx --format=pretty ./src",
    "lint:prettier": "prettier -c --write \"src/**/*\" --end-of-line auto",
    "openapi": "max openapi",
    "playwright": "playwright install && playwright test",
    "prepare": "husky install",
    "prettier": "prettier -c --write \"src/**/*\"",
    "serve": "umi-serve",
    "start": "cross-env REACT_APP_ENV=dev UMI_ENV=dev max dev",
    "start:dev": "cross-env PORT=8072 MOCK=none REACT_APP_ENV=dev UMI_ENV=dev max dev",
    "start:no-mock": "cross-env MOCK=none UMI_ENV=dev max dev",
    "start:pre": "cross-env REACT_APP_ENV=pre UMI_ENV=dev max dev",
    "start:test": "cross-env REACT_APP_ENV=test MOCK=none UMI_ENV=dev max dev",
    "test:e2e": "node ./tests/run-tests.js",
    "tsc": "tsc --noEmit"
  },
  "lint-staged": {
    "**/*.{js,jsx,ts,tsx}": "npm run lint-staged:js",
    "**/*.{js,jsx,tsx,ts,less,md,json}": [
      "prettier --write"
    ]
  },
  "browserslist": [
    "> 1%",
    "last 2 versions",
    "not ie <= 10"
  ],
  "dependencies": {
    "@ant-design/icons": "^4.7.0",
    "@ant-design/pro-components": "^2.0.0",
    "@types/node": "^18.14.2",
    "@umijs/route-utils": "^2.1.3",
    "ahooks": "^3.7.2",
    "antd": "^4.20.0",
    "classnames": "^2.3.0",
    "fs-extra": "^11.1.0",
    "lodash": "^4.17.21",
    "moment": "^2.29.0",
    "omit.js": "^2.0.2",
    "rc-menu": "^9.1.0",
    "rc-util": "^5.16.0",
    "rc-virtual-list": "^3.4.11",
    "react": "^17.0.0",
    "react-dev-inspector": "^1.7.0",
    "react-dom": "^17.0.0",
    "react-helmet-async": "^1.2.0",
    "react-lazyload": "^3.2.0"
  },
  "devDependencies": {
    "@ant-design/pro-cli": "^2.1.0",
    "@playwright/test": "^1.17.0",
    "@types/classnames": "^2.3.1",
    "@types/express": "^4.17.0",
    "@types/history": "^4.7.0",
    "@types/lodash": "^4.14.0",
    "@types/react": "^17.0.0",
    "@types/react-dom": "^17.0.0",
    "@types/react-helmet": "^6.1.0",
    "@umijs/fabric": "^2.11.1",
    "@umijs/max": "^4.0.0-rc.22",
    "@umijs/openapi": "^1.3.0",
    "cross-env": "^7.0.0",
    "cross-port-killer": "^1.3.0",
    "detect-installer": "^1.0.0",
    "eslint": "^7.32.0",
    "gh-pages": "^3.2.0",
    "husky": "^7.0.4",
    "jszip": "^3.10.1",
    "lint-staged": "^10.0.0",
    "mockjs": "^1.1.0",
    "prettier": "^2.5.0",
    "swagger-ui-dist": "^4.12.0",
    "typescript": "^4.5.0",
    "umi-presets-pro": "^1.0.1",
    "umi-serve": "^1.9.10"
  },
  "engines": {
    "node": ">=12.0.0"
  },
  "__npminstall_done": false
}
`

**Comment by fz6m** [comment]
更新下最新的 `@umijs/max` 版本看看，还不行的话需要提供一个最小复现
